### PR TITLE
x11-misc/rofi-calc: new ebuild

### PR DESCRIPTION
**Pull request by hallabro**
rofi-calc is a rofi (x11-misc/rofi) plugin that uses libqalculate's
qalc to parse natural language input and provide results.

Bug: https://bugs.gentoo.org/704864
Package-Manager: Portage-2.3.84, Repoman-2.3.20
Signed-off-by: Robin Hallabro-Kokko <robin@hallabro.nu>

### CODE REVIEW

File: x11-misc/rofi-calc/rofi-calc-1.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+HOMEPAGE="https://github.com/svenstaro/rofi-calc"
+SRC_URI="https://github.com/svenstaro/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	>=x11-misc/rofi-1.5
```
Comment (juippis) [inline x11-misc/rofi-calc/rofi-calc-1.5.ebuild]: This version restriction is always satisfied in ::gentoo and can be removed.

File: x11-misc/rofi-calc/rofi-calc-1.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils autotools
```
Comment (juippis) [inline x11-misc/rofi-calc/rofi-calc-1.5.ebuild]: `eutils` doesnt seem to be used in this ebuild?

Same comments apply to -9999.

File: x11-misc/rofi-calc/rofi-calc-1.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+DESCRIPTION="Do live calculations in rofi!"
+HOMEPAGE="https://github.com/svenstaro/rofi-calc"
+SRC_URI="https://github.com/svenstaro/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (juippis) [inline x11-misc/rofi-calc/rofi-calc-1.5.ebuild]: Please open the `${PN}` to plaintext, makes copypasting easier and it's not going to change.

**Comment by hallabro** [comment]
Thanks for the feedback, @juippis. Should be fixed now.

**Comment by juippis** [comment]
ad76d4caa79d2b23bd258ea18f065e2631c73cf1

**Pull request closed without merging**